### DISCUSSION TITLE
Free-RASP-Cordova plugin can be bypass using frida script

### DISCUSSION THREAD

**Discussion by dshark95** [talsec/Free-RASP-Cordova#8] [Q&A] [answered]
Hi, my company app have been using Free-RASP-Cordova plugin for awhile. Recently company security team have found a way to bypass the talsec execute method by using frida script, just wanted to ask is there anything I can do to enhance the app to prevent the execution of the frida script?

**Comment by SirionRazzer** [comment, marked as answer]
Hi @dshark95 , Frida bypasses were addressed in the latest release of freeRASP, discussed here [this issue](https://github.com/talsec/Free-RASP-Flutter/issues/87#issuecomment-1910264016). It should prevent simpler types of hooking bypasses. I hope this helps 🙂. I will close this discussion, but feel free to open a new one if you want to discuss it further.

**Reply by dshark95** [reply]
@SirionRazzer thanks for the update, will try it out.
